build/Build.cs

```diff
@@ -92,46 +101,29 @@ class Build : NukeBuild
                 .SetProject(Solution)
                 .SetConfiguration(Configuration)
                 .SetOutputDirectory(ArtifactsDirectory)
-                .EnableNoBuild()
+                .SetNoBuild(true)
                 .AddProperty("Version", OctoVersionInfo.FullSemVer)
             );
         });
 
-    Target Publish => _ => _
-        .OnlyWhenStatic(() => IsServerBuild)
-        .DependsOn(Pack)
-        .Executes(() =>
-        {
-            var packageName = Solution.Octodiff.Name;
-            NuGetTasks.NuGetPush(_ => _
-                .SetTargetPath(ArtifactsDirectory / $"{packageName}.{OctoVersionInfo.FullSemVer}.nupkg")
-                .SetSource("https://f.feedz.io/octopus-deploy/dependencies/nuget")
-                .SetApiKey(FeedzIoApiKey)
-            );
-            if (string.IsNullOrWhiteSpace(OctoVersionInfo.PreReleaseTagWithDash))
-            {
-                NuGetTasks.NuGetPush(_ => _
-                    .SetTargetPath(ArtifactsDirectory / $"{packageName}.{OctoVersionInfo.FullSemVer}.nupkg")
-                    .SetSource("https://www.nuget.org/api/v2/package")
-                    .SetApiKey(NuGetApiKey));
-            }
-        });
-
     Target CopyToLocalPackages => _ => _
         .OnlyWhenStatic(() => IsLocalBuild)
         .DependsOn(Pack)
         .Executes(() =>
         {
-            GlobFiles(ArtifactsDirectory, $"*.{OctoVersionInfo.FullSemVer}.nupkg")
-                .ForEach(x => CopyFileToDirectory(x, LocalPackagesDirectory, FileExistsPolicy.Overwrite));
+            EnsureExistingDirectory(LocalPackagesDirectory);
+            CopyFileToDirectory(ArtifactsDirectory / $"{Solution.Name}.{OctoVersionInfo.FullSemVer}.nupkg",
+                LocalPackagesDirectory, FileExistsPolicy.Overwrite);
         });
 
+    Target Default => _ => _
+        .DependsOn(Pack)
+        .DependsOn(CopyToLocalPackages);
```

Comment:
`CopyToLocalPackages` may not be needed anymore; I think it was a step before publish that we're not doing anymore.